src/components/media/video/video.tsx

```diff
@@ -42,20 +41,20 @@ export default function Video(props: VideoFormat) {
   const isMobile = useMediaQuery(theme.breakpoints.down('md'))
   const inActivityTimeout = 3000 // Hide controls after 3 seconds of inactivity
   useEffect(() => {
-    const loadingErrorMessage = 'The video resource has failed to load'
-    const stalledErrorMessage = 'Failed to fetch data, but trying'
+    const onErrorMessage = 'The video resource has failed to load'
+    const onStalledMessage = 'Failed to fetch data, but trying'
```

Comment:
```suggestion
    const errorMessage = 'Failed to load the video.'
    const stalledErrorMessage = 'Failed to fetch data, but trying.'
```
If these are only used once, I think they can defined where they used instead of globally in useEffect. 
Alternatively, if these are used in multiple places, the constants can be moved to the top of the component or outside the component definition.